### DISCUSSION TITLE
Support for blazingsql

### DISCUSSION THREAD

**Discussion by mctouch** [apache/superset#18557] [Ideas]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

support for https://blazingsql.com/ an in GPU SQL database

https://github.com/BlazingDB

**Describe alternatives you've considered**

**Additional context**

**Comment by junlincc** [comment]
cc @srinify ~

**Comment by mistercrunch** [comment]
For Superset to work with any SQL-speaking database, it requires for that database to have a DBAPI-compatible driver as well as a SQLAlchemy dialect.

Those are well documented Python standards that aren't too difficult to implement. The dialect is generally super easy, and the DBAPI driver is kind of a must for the database to work with the Python ecosystem.

**Comment by srinify** [comment]
@mctouch hey there, I'm happy to collaborate on this and help get this working. Can you investigate if blazingsql has a dbapi-compatible driver + a SQLAlchemy dialect?

**Comment by srinify** [comment]
hey @mctouch just thought I'd follow up!

**Comment by amirsani** [comment]
Just curious if there has been any progress on this.

**Comment by MaxPowerWasTaken** [comment]
I would also love this, left a feedback/comment at at https://docs.blazingdb.com/docs that I wasn't able to find a dbapi driver and sqlalchemy dialect and referenced this issue, hopefully someone from BlazingSQL will see it and pop in...
